tasks/README.md

```diff
@@ -0,0 +1,96 @@
+# Tasks
+
+## Task object
+
+```
+{
+  "type":"Dev | Group",
+  "links": [
+    <link1>,
+    <link2>
+  ],
+  "endsOn":"<end date>",
+  "startedOn":"<start date>",
+  "status": "Active",
+  "ownerId":"<app owner user id>",
+  "percentCompleted":10%,
+  "dependsOn": [
+    <tasksList>
+  ],
+  "participants": [list of users participating in this tasks. For 'group' type only],
+  "completionAward": { gold: 3, bronze: 300 },
+  "lossRate": { gold: 1 }  // Loss per day of overshoot after deadline
+}
+```
+
+## **Requests**
+
+|               Route                |    Description    |
+| :--------------------------------: | :---------------: |
+|      [GET /tasks](#get-tasks)      | Returns all tasks |
+|     [POST /tasks](#post-tasks)     | Creates new task  |
+| [PATCH /tasks/:id](#patch-tasksid) |   Updates tasks   |
+
+## **GET /tasks**
+
+> returns all the tasks
+
+- **Success Response:**
+- **Code:** 200
+  - **Content:**
+
+```
+{
+  message: 'Tasks returned successfully!'
+  tasks: [
+           {<task_object>},
+           {<task_object>}
+         ]
+}
+```
+
+- **Error Response:**
+  - **Code:** 503
+    - **Content:** `{ 'statusCode': 503, 'error': 'Service Unavailable', 'message': 'Something went wrong please contact admin' }`
```

Comment:
Why indent?

Reply:
To make it more readable

![image](https://user-images.githubusercontent.com/29303618/104115102-a7e15f00-5331-11eb-94ca-baf0b0d7790a.png)


please let me know if we need to remove it.